examples/hotrod/pkg/tracing/init.go

```diff
@@ -43,9 +44,28 @@ import (
 
 var once sync.Once
 
-// Init initializes OpenTelemetry SDK and uses OTel-OpenTracing Bridge
+// InitOTEL initializes OpenTelemetry SDK
+// to return an Otel tracer.
+func InitOTEL(serviceName string, exporterType string, metricsFactory metrics.Factory, logger log.Factory) trace.Tracer {
+	_, oteltp := initBOTH(serviceName, exporterType, metricsFactory, logger)
+	otel.SetTracerProvider(oteltp)
+
+	logger.Bg().Debug("Added OTEL tracer", zap.String("service-name", serviceName))
+	return oteltp.Tracer(serviceName)
+}
+
+// InitOP initializes OTel-OpenTracing Bridge
 // to return an OpenTracing-compatible tracer.
 func Init(serviceName string, exporterType string, metricsFactory metrics.Factory, logger log.Factory) opentracing.Tracer {
+	optracer, _ := initBOTH(serviceName, exporterType, metricsFactory, logger)
+
+	logger.Bg().Debug("created OTEL->OT bridge", zap.String("service-name", serviceName))
```

Comment:
```suggestion
	logger.Bg().Debug("Created OTEL->OT bridge", zap.String("service-name", serviceName))
```
Usually the log statements start with a capital letter, to read like full sentences. But the error messages are traditionally not capitalized.